docs/source/en/api/pipelines/text_to_video_zero.md

```diff
@@ -146,19 +143,19 @@ To generate a video from prompt with additional pose control
 
     prompt = "Darth Vader dancing in a desert"
     result = pipe(prompt=[prompt] * len(pose_images), image=pose_images, latents=latents).images
-    imageio.mimsave("video.mp4", result, fps=4)
+    result_np = np.concatenate([np.array(r)[None, :, :, :] for r in result])
+    export_to_video(result_np, "video.mp4")
```

Comment:
Can you also confirm here (and other places where `imageio.mimsave("edited_video.mp4", result, fps=4)` is replaced) whether you're able to generate a meaningful video?